### PR TITLE
Add isalzman burbank recommendations file

### LINKED ISSUE #7: Create a file for isalzman Burbank, CA
- [x] Create a branch
- [x] Add commit
- [x] Open a PR
- [x] Have a discussion
- [x] Seek approval
- [x] Merge PR

### PR DESCRIPTION
**Pull request by ISalzman**
closes #7

### CODE REVIEW

**Comment by QAscoop** [comment]
🛠

File: isalzman-burbank.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Burbank, CA
+
+### Places to Eat
+- Roach Coach
+- Cafe de Crew
+
+### Places to Go
+- Cast & Crew
```
Comment (kulkarni2u) [inline isalzman-burbank.md]: Nice you want to visit Office 👍

File: isalzman-burbank.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# Burbank, CA
+
+### Places to Eat
+
+### Places to Go
```
Comment (QAscoop) [inline isalzman-burbank.md]: Parking lot?

**Review by QAscoop — COMMENTED** [review]
Hmmm

File: isalzman-burbank.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# Burbank, CA
+
+### Places to Eat
+- Roach Coach
+- Cafe de Crew
+- Starbucks
```
Comment (kulkarni2u) [inline isalzman-burbank.md:6]: 👍

File: isalzman-burbank.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+- BevMo
+
+### Places to Go
+- Cast & Crew
+- Bob Hope Airport
+- Ikea
```
Comment (kulkarni2u) [inline isalzman-burbank.md:12]: That is something new! 🏆

**Review by kulkarni2u — APPROVED** [review]

**Pull request merged**